x/clp/keeper/keeper.go

```diff
@@ -12,7 +12,7 @@ import (
 // Keeper of the clp store
 type Keeper struct {
 	storeKey   sdk.StoreKey
-	cdc        *codec.Codec
+	Cdc        *codec.Codec
```

Comment:
I think it would be a better Idea , to create getter for this isntead of exposing it directly

Reply:
Fair point, I'll fix this.